### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by hemanoel**
I have updated the README and usage files to fix typos, improve text clarity in some points, and provide more details about how to use phaseimpute.

## PR checklist
- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/phaseimpute/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/phaseimpute _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core pipelines lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,23 +20,18 @@
...
+| <img src="docs/images/metro/MetroMap.svg" alt="metromap" width="800"/> | - **Check chromosome**: Validates the presence of contigs from the map file in the VCF, ensuring data compatibility for further processing <br> - **Panel preparation**: Perfoms the phasing, QC, variant filtering, variant annotation of the reference panel <br> - **Imputation**: Imputes missing genotypes in the target dataset using the reference panel <br> - **Simulate**: Generates simulated datasets from high-quality target data for testing and validation purposes. <br> - **Concordance**: Evaluates the accuracy of imputation by comparing the imputed data against a truth dataset. |
```
Comment (LouisLeNezet) [inline README.md:27]:
```suggestion
| <img src="docs/images/metro/MetroMap.svg" alt="metromap" width="800"/> | - **Check chromosomes names**: Validates the presence of the different contigs in all variants and alignment files, ensuring data compatibility for further processing <br> - **Panel preparation**: Perfoms the phasing, QC, variant filtering, variant annotation of the reference panel <br> - **Imputation**: Imputes genotypes in the target dataset using the reference panel <br> - **Simulate**: Generates simulated datasets from high-quality target data for testing and validation purposes. <br> - **Concordance**: Evaluates the accuracy of imputation by comparing the imputed data against a truth dataset. |
```

File: docs/usage.md [hunk 2]
```diff
@@ -6,17 +6,19 @@
 
 ## Introduction
 
+The **nf-core/phaseimpute** pipeline is designed to perform the phasing and imputation techniques. Some key functionalities include chromosome checking, panel preparation, imputation, simulation, and concordance.
```
Comment (LouisLeNezet) [inline docs/usage.md:9]:
```suggestion
The **nf-core/phaseimpute** pipeline is designed to perform genomic phasing and imputation techniques. Some key functionalities include chromosome checking, panel preparation, imputation, simulation, and concordance.
```

File: docs/usage.md [hunk 3]
```diff
@@ -128,14 +130,32 @@ or you can specify a custom genome using:
...
+    --input samplesheet.csv \
+    --steps impute \
+    --chunks chunks.csv \
+    --posfile posfile_legend.csv \
+    --outdir results \
+    --genome GRCh38 \
+    --panel panel.csv \
+    --tools glimpse1 \
+    -profile docker
+
```
Comment (LouisLeNezet) [inline docs/usage.md:146]:
```suggestion
```

File: docs/usage.md [hunk 4]
```diff
@@ -128,14 +130,32 @@ or you can specify a custom genome using:
...
 
 ```bash
 nextflow run nf-core/phaseimpute \
     --input samplesheet.csv \
-    --steps panelprep,impute
+    --steps panelprep,impute \
     --outdir results \
-    --genome GRCh37 \
+    --genome GRCh38 \
+    --panel
```
Comment (LouisLeNezet) [inline docs/usage.md:157]: File missing here ?

**Review by LouisLeNezet — CHANGES_REQUESTED** [review]
You can add yourself in the Changelog and in the readme.

**Comment by atrigila** [comment]
Great job! Could you also fill in the changelog? :)

**Review by LouisLeNezet — APPROVED** [review]

**Pull request merged**